nimbus/src/enrollment.rs

```diff
@@ -435,8 +440,9 @@ pub enum EnrollmentStatus {
 }
 
 impl EnrollmentStatus {
-    pub fn new_enrolled(reason: EnrolledReason, branch: &str) -> Self {
+    pub fn new_enrolled(reason: EnrolledReason, branch: &str, feature_id: &str) -> Self {
```

Comment:
This is temporary, isn't it? Perhaps a comment would be useful, or pass in a `&Branch` that the `fn` can dig out the slug and feature ids from the branch.

Reply:
Duly commented.